### PR TITLE
Add support for emoji_changed event

### PR DESCRIPTION
**Pull request by ToshihitoKon**
added support EventAPI `emoji_changed` event

https://api.slack.com/events/emoji_changed

### CODE REVIEW

File: slackevents/inner_events.go [hunk 1]
```diff
@@ -182,6 +182,24 @@ type TokensRevokedEvent struct {
 	Tokens tokens `json:"tokens"`
 }
 
+// EmojiChangedEvent is the event of custom emoji has been added or changed
+type EmojiChangedEvent struct {
+	Type           string      `json:"type"`
+	Subtype        string      `json:"subtype"`
+	Value          string      `json:"value"`
```
Comment (kanata2) [inline slackevents/inner_events.go]: This field is an empty when removed emoji

Comment (kanata2) [reply slackevents/inner_events.go:189]: Does Slack send us to different errors for SocketMode and RTM??

Comment (ToshihitoKon) [reply slackevents/inner_events.go]: thanks for review and fixed it!

**Review by kanata2 — APPROVED** [review]
👍

**Pull request merged**